pontis-ui/src/components/BigButton.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import classNames from "classnames";
+import { ChevronRightIcon } from "@heroicons/react/outline";
+
+interface BigButtonProps {
+  name: string;
+  isActive: boolean;
+  handleMouseEnter: () => void;
+  href: string;
+}
+
+const BigButton: React.FC<BigButtonProps> = ({
+  name,
+  isActive,
+  handleMouseEnter,
+  href,
+}) => {
+  return (
+    <div className="h-48 w-full sm:h-[300px] lg:h-[500px]">
+      <a
+        className={classNames(
+          "sm:text-4-xl flex h-full cursor-pointer flex-col justify-end border-[10px] border-slate-900 p-8 text-3xl font-bold text-slate-900 transition-all duration-300 ease-in motion-reduce:transition-none sm:border-0 sm:p-[84px] lg:text-5xl",
+          { "sm:border-[20px] sm:p-16": isActive }
+        )}
+        onMouseEnter={handleMouseEnter}
+        href={href}
```

Comment:
target black in my opinion. Users will frequently want to come back to our page, and closing a tab is easier than navigating some number of pages back

Reply:
Check out [this article](https://css-tricks.com/use-target_blank/). I think this needs more discussion.

Reply:
I hear the points that article is making and still disagree with them -- I'm in favor of target="_blank" for many of the reasons they call "bad reasons" (mainly easier to navigate back, given we're redirecting to an external page which is likely not user's end destination, they are just exploring). If you feel strongly we can leave it out